reportportal_client/service.py

```diff
@@ -42,14 +43,15 @@ def _convert_string(value):
     return str(value)
 
 
-def _list_to_payload(dictionary):
+def _dict_to_payload(dictionary):
     """Convert dict to list of dicts.
 
     :param dictionary: initial dict
     :return list: list of dicts
     """
+    system = dictionary.pop('system', False)
```

Comment:
Use either single quotes or doubles within the whole module.

Reply:
it doesnt matter, pep8 allow to use single and double quotes